moff.py

```diff
@@ -40,700 +39,749 @@
-		#log.info("\t LW_BOUND window  %4.4f", time_w - temp_w)
-		#log.info("\t UP_BOUND window %4.4f", time_w + temp_w)
-		#log.info("\t WARNINGS: Peak not detected  Xic shape %r ", data_xic[(data_xic['rt'] > (time_w - temp_w)) & (data_xic['rt'] < (time_w + temp_w))].shape[0])
-		return  pd.Series({'intensity': -1, 'rt_peak': -1,
-				   'lwhm':-1,
-					'rwhm':-1,
-					'5p_noise':-1,
-					'10p_noise':-1,
-					'SNR':-1,
-					'log_L_R':-1,
-					'log_int':-1})
-	pnoise_5 = np.percentile(data_xic[(data_xic['rt'] > (time_w - h_rt_w )) & (data_xic['rt'] < (time_w + h_rt_w ))]['intensity'], 5 )
-	pnoise_10 = np.percentile( data_xic[(data_xic['rt'] > (time_w - h_rt_w )) & (data_xic['rt'] < (time_w + h_rt_w)  )]['intensity'], 10)
-	# find the lwhm and rwhm
-	time_point = compute_log_LR(data_xic, pos_p[0], val_max, 0.5)
-	if (time_point[0]*time_point[1] == 1) or (time_point[0]*time_point[1] < 0):
-		# Try a second time FWHM  computation with 0.7 * max intensity
-		time_point =  compute_log_LR (data_xic,pos_p[0],val_max,0.7)
-
-	if time_point[0]== -1 or  time_point[1] ==-1:
-		# keep the shape measure to -1 in case on txo point are -1
-		log_L_R=-1
-	else:
-			log_L_R= np.log2(abs( time_w  - time_point[0]) / abs( time_w - time_point[1]))
-	
-	if (pnoise_5 == 0 and pnoise_10 > 0):
-				SNR  = 20 * np.log10(data_xic.ix[pos_p, 1].values / pnoise_10)
-	else:
-		if pnoise_5 != 0:
-				SNR = 20 * np.log10(data_xic.ix[pos_p, 1].values / pnoise_5)
-		else:
-				log.info('\t 5 percentile is %4.4f (added 0.5)', pnoise_5)
-				SNR = 20 * np.log10(data_xic.ix[pos_p, 1].values / (pnoise_5 +0.5))
-	
-	return pd.Series({'intensity': val_max[0], 'rt_peak': data_xic.ix[pos_p, 0].values[0] * 60,
-			   'lwhm': time_point[0] ,
-				'rwhm': time_point[1] ,
-				'5p_noise': pnoise_5,
-				'10p_noise':pnoise_10,
-				'SNR':SNR[0],
-				'log_L_R': log_L_R,
-				'log_int': np.log2(val_max)[0] })
+            # standar cases rt must be in second
+            time_w = time_w / 60
+    # print time_w, x['rt'] , moff_pride_flag, rt_match_peak,time_w, s_w,s_w_match
+    if mbr_flag == 0:
+        # log.info('peptide at line %i -->  MZ: %4.4f RT: %4.4f',(offset_index +c +2), x['mz'], time_w)
+        temp_w = s_w
+    else:
+                    # row['matched'])
+        if x['matched'] == 1:
+            temp_w = s_w_match
+
+        else:
+            temp_w = s_w
+    if data_xic[(data_xic['rt'] > (time_w - temp_w)) & (data_xic['rt'] < (time_w + temp_w))].shape[0] >= 1:
+        # data_xic[(data_xic['rt'] > (time_w - temp_w)) & (data_xic['rt'] < (time_w + temp_w))].to_csv('thermo_testXIC_'+str(c)+'.txt',index=False,sep='\t')
+        ind_v = data_xic.index
+        pp = data_xic[data_xic["intensity"] == data_xic[(data_xic['rt'] > (
+            time_w - temp_w)) & (data_xic['rt'] < (time_w + temp_w))]['intensity'].max()].index
+        pos_p = ind_v[pp]
+        if pos_p.values.shape[0] > 1:
+            print 'error'
```

Comment:
this should be a log.warning or something similar.

Reply:
fixed